.github/workflows/deploy.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           # Store current git hash and date in files
           mkdir -p buildinfo
           echo "${GITHUB_SHA}" > buildinfo/SHA
-          printf '%(%Y-%m-%dT%H:%M:%SZ)T' > buildinfo/DATE
+          printf '%(%Y-%m-%dT%H:%M:%S%z)T' > buildinfo/DATE
```

Comment:
I think this is safer and will just use +or- 0000 if the build machine is on UTC time